validator/proposer/service.go

```diff
@@ -95,13 +173,19 @@ func (p *Proposer) run(done <-chan struct{}, client pb.ProposerServiceClient) {
 			}
 			latestBlockHash := blake2b.Sum512(data)
 
+			// To prevent any unaccounted attestations from being added.
+			p.mutex.Lock()
+
+			agSig := p.AggregateAllSignatures(p.pendingAttestation)
+			bitmask := p.GenerateBitmask(p.pendingAttestation)
```

Comment:
Slightly out of scope for this PR, this touch bases on proposer responsibility. Need @rauljordan to take a look to make sure there's no duplication of work on his end

Reply:
Yeah I removed what I previously implemented for those two functions, so currently those functions are just stubs. Whoever is doing proposer responsibility can just implement it themselves